src/Games/NexusMods.Games.AdvancedInstaller.UI/Content/BodyView.axaml

```diff
@@ -62,9 +62,7 @@
             <Grid ColumnDefinitions="*, Auto, *" x:Name="CentralGrid">
                 <!-- Mod Content -->
                 <Border Classes="Rounded-lg Mid" Grid.Column="0" x:Name="ModContentBorder">
-                    <ScrollViewer x:Name="ModContentScrollViewer">
-                        <reactiveUi:ViewModelViewHost x:Name="ModContentSectionViewHost" />
-                    </ScrollViewer>
+                    <reactiveUi:ViewModelViewHost x:Name="ModContentSectionViewHost" />
```

Comment:
I checked in with Design, the `ScrollViewer` here is intentional. The section header is supposed to scroll up to leave space to the tree. 
You can either revert the change or deal with this in a separate PR if you want investigate virtualization issues.

Reply:
Moved to #1048